k2/csrc/math.h

```diff
@@ -124,6 +124,10 @@ inline std::istream &operator >>(std::istream &is, InputFixer<double> &f) {
   return is;
 }
 
+// Return the seed that can be used for random generators.
+// It reads the environment variable to get the seed.
```

Comment:
maybe better to say, it reads the environment variable `K2_SEED` to... or `0` if `K2_SEED` is not set?